Source/Core/Core/PowerPC/JitArm64/JitArm64_LoadStore.cpp

```diff
@@ -775,10 +775,11 @@ void JitArm64::dcbx(UGeckoInstruction inst)
   m_float_emit.ABI_PushRegisters(fprs_to_push, WA);
 
   // The function call arguments are already in the correct registers
```

Comment:
```suggestion
  // The first two function call arguments are already in the correct registers
```

(Or something equivalent, like "W0 and W1 are already set to the correct values")